.github/workflows/buid_and_release.yml

```diff
@@ -6,48 +6,41 @@ name: Build and release
 on:
   schedule:
     - cron: "0 0 * * 0"  # weekly on Sunday at midnight
-#    - cron: '0 0 * * *'  # every day at midnight
-#    - cron: "0 0 1 * *"  # monthly
   workflow_dispatch:
 
 jobs:
   build_and_release:
     runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.4.1
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.10
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.10'
     - name: Install
       run: |
-        python -m pip install --upgrade pip
         make install
     - name: Create .env file
       run: |
-        echo -en "API_KEY=${{ secrets.API_KEY }}" > .env
+        echo "API_KEY=${{ secrets.API_KEY }}" > .env
```

Comment:
A bugfix. For some reason `echo -en` wasn't working for me after switching over to the ODK container.